cmd/root.go

```diff
@@ -45,6 +45,6 @@ func Execute() {
 }
 
 func init() {
-	RootCmd.PersistentFlags().StringVar(&cfgFile, "config", "", "config file (default is $HOME/.mctl.yml)")
+	RootCmd.PersistentFlags().StringVar(&cfgFile, "config", "", "config file (default is $XDG_CACHE_DIR/mctl/config.yml)")
```

Comment:
Shouldn't this be `$XDG_CONFIG_HOME` instead? According to the spec https://specifications.freedesktop.org/basedir-spec/basedir-spec-latest.html, it's where the user config files should be located. This env var wouldn't be used on a Mac environment, AFAICT.

Reply:
1000% right... will fix

Reply:
> This env var wouldn't be used on a Mac environment, AFAICT.

The go package I'm using here basically does the right thing for other platforms too:

#### Unix-like operating systems
||Unix|macOS|Plan 9|
|--|--|--|--|
|XDG_CONFIG_HOME|~/.config|~/Library/Application Support|$home/lib|

#### Microsoft Windows
||Known Folder(s)|Fallback(s)|
|--|--|--|
|XDG_CONFIG_HOME|LocalAppData|%LOCALAPPDATA%|

Reply:
Not quite right still, diff shows `$XDG_CONFIG_DIR` while it should be `$XDG_CONFIG_HOME`. 

Good to know it works everywhere! Thanks for the fixes!

Reply:
thats what I get when I sed w/o thinking :/

Reply:
@diogomatsubara ptal again, thanks!